app/controllers/api/v1/accounts/contacts_controller.rb

```diff
@@ -1,4 +1,4 @@
-class Api::V1::Accounts::ContactsController < Api::V1::Accounts::BaseController
+class Api::V1::Accounts::ContactsController < Api::V1::Accounts::BaseController # rubocop:disable Metrics/ClassLength
```

Comment:
let's add the rubocop disable to `rubocop.yml` so that we will have a list at one place which we can fix later

https://github.com/chatwoot/chatwoot/blob/a4c87f2052d9a668f9cb9b20e3f1bb8ec6c80f4f/.rubocop.yml#L10